src/pages/DisbursementDraftDetails.tsx

```diff
@@ -231,7 +244,8 @@ export const DisbursementDraftDetails = () => {
         }}
         isDraftDisabled={!isCsvFileUpdated}
         isSubmitDisabled={
-          !(Boolean(draftDetails) && Boolean(csvFile) && canUserSubmit)
+          !(Boolean(draftDetails) && Boolean(csvFile) && canUserSubmit) ||
+          handleCalculateFutureBalance() < 0
```

Comment:
We're calling the `handleCalculateFutureBalance()` function multiple times. How do you feel about assigning it to a variable and using that variable instead?